workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Readers/BlittableReaderBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    public abstract class BlittableReaderBase<TSpatialComponentData, TComponentUpdate>
+        : ReaderBase<TSpatialComponentData, TComponentUpdate>
+        where TSpatialComponentData : struct, ISpatialComponentData, IComponentData
+        where TComponentUpdate : ISpatialComponentUpdate<TSpatialComponentData>
```

Comment:
Again, is this second generic constraint required? It doesn't actually provide any extra constraints?